messenger.java

```diff
@@ -18,7 +27,7 @@
 	{
 
 		final JLabel message=new JLabel("Please enter ip to connect");
-		final JTextField ipui=new JTextField();
+		final JTextField ipui=new JTextField("127.0.0.1");
```

Comment:
it was getting increasingly tedious to type that again and again, so added that as a default!